### ISSUE TITLE
Hub Timeout?

### ISSUE THREAD

**Issue by JangoBritt** [KraigM/homebridge-harmonyhub#54]
So I'm getting an issue with my homebridge now. It won't boot up occasionally, and I get the message below.

errorhub { [Error: connect ETIMEDOUT 192.168.1.232:5222]
  code: 'ETIMEDOUT',
  errno: 'ETIMEDOUT',
  syscall: 'connect',
  address: '192.168.1.232',
  port: 5222 }

I am able to ping that address just fine, and log on with my iPhone app.

I did receive a max clients error a few days ago, so this might be related to #31 and #51

**Comment by mgoblue01** [comment]
Same problem here.  Any solution?

**Comment by joq3** [comment]
Same problem! Has anyone solved this?

**Comment by tomto147** [comment]
I was getting the same error message. In my opinion it is caused by the computer going to sleep mode after some time. I used to use this plugin on my MAC. Now I have installed everything on Raspberry Pi and no problems for the last 3 days. Also I was getting the same problem when the harmony hub was unreachable because of wifi down or power down to the hub itself. Even if the hub was unreachable for few seconds I got this message.
